### PR TITLE
[examples][tests] Update fixture test names to be concretely named

### PR DESCRIPTION
**Pull request by trek**
Working on resolving some flakey tests and the test descriptions are all `should deploy` which made tracking down the specific test a bit of a hunt. This should make identifying them easier for any future needs.

### CODE REVIEW

**Review by Ethan-Arrowood — DISMISSED** [review]

**Review by EndangeredMassa — DISMISSED** [review]

File: examples/__tests__/integration/hexo.test.ts [hunk 1]
```diff
@@ -1,5 +1,4 @@
 import { deployExample } from '../test-utils';
-it('should deploy', async () => {
+it('[examples] should deploy ', async () => {
```
Comment (cb1kenobi) [inline examples/__tests__/integration/hexo.test.ts]: Missing `hexo`?

File: examples/__tests__/integration/nextjs.test.ts [hunk 2]
```diff
@@ -1,5 +1,4 @@
 import { deployExample } from '../test-utils';
-it('should deploy', async () => {
+it('[examples] should deploy ', async () => {
```
Comment (cb1kenobi) [inline examples/__tests__/integration/nextjs.test.ts]: Missing `nextjs`?

File: examples/__tests__/integration/preact.test.ts [hunk 3]
```diff
@@ -1,5 +1,4 @@
 import { deployExample } from '../test-utils';
-it('should deploy', async () => {
-  await deployExample(__filename);
+it('[examples] should deploy preact', async () => {
+  await deployExample('preact.test');
```
Comment (cb1kenobi) [inline examples/__tests__/integration/preact.test.ts]:
```suggestion
  await deployExample('preact');
```

File: examples/__tests__/integration/hexo.test.ts (hunk 1 above)
Comment (trek) [reply examples/__tests__/integration/hexo.test.ts]: lolz, losing my mind 
CleanShot 2023-10-24 at 11 22 55@2x

until I saw 

CleanShot 2023-10-24 at 11 23 10@2x

**Review by cb1kenobi — APPROVED** [review]

File: examples/__tests__/integration/hexo.test.ts (hunk 1 above)
Comment (cb1kenobi) [reply examples/__tests__/integration/hexo.test.ts]: Ha!

**Review by TooTallNate — APPROVED** [review]

**Pull request merged**